en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-300.md

```diff
@@ -1797,6 +1797,14 @@ Follow the instructions below to move all the existing API Manager configuration
         ```
 
 5.  Copy the keystores (i.e., `client-truststore.jks`, `wso2cabon.jks` and any other custom JKS) used in the previous version and replace the existing keystores in the `<API-M_3.0.0_HOME>/repository/resources/security` directory.
+    
+    **Note:** From API Manager 3.0.0 onwards, keystores could be configured as follows.
+    
```

Comment:
Please note that the following comments are applicable for all the changes in this commit.